### DISCUSSION TITLE
New feature for FluxGPT app

### DISCUSSION THREAD

**Discussion by minhcasi** [inspireui/support#5743] [New Feature/Ideas Requests]
1- Create a global API for admin to be able to create various plans and offer services based on the number of words/characters/images etc...  and other parameters configurable from the admin dashboard.
2- Prompt/template categories module to be configured/created from the admin dashboard.
3- Prompt / template - admin to be able to create flow-based prompt templates from dashboard. (Using variables in the prompt)
4- Integrate Stability.ai for superior AI image generation.
5- Team module to allow team members collaborate.

**Comment by rocketdigitalai** [comment]
1 and 4 are very important in my opinion.

**Reply by rpaix** [reply]
I think they are all important. Quality of prompts and the ability for the admin to generate them is a deal breaker. Also collaborative team environment is absolutely necessary, otherwise why would one choose this app?? they go directly to openai!
